core/src/main/scala/kafka/server/ReplicaAlterLogDirsThread.scala

```diff
@@ -105,7 +105,7 @@ class ReplicaAlterLogDirsThread(name: String,
       topicId <- partition.topicId
       directoryId <- partition.logDirectoryId()
       topicIdPartition = new TopicIdPartition(topicId, topicPartition.partition())
-    } directoryEventHandler.handleAssignment(topicIdPartition, directoryId, () => ())
+    } directoryEventHandler.handleAssignment(topicIdPartition, directoryId, "Reverting queued dir reassignment mid future replica promotion", () => ())
```

Comment:
What does "mid future" mean here?

Reply:
These aren't the words we'd want to couple together. I meant "mid [future replica promotion]". i.e. in the middle of the process of promoting a future replica. Perhaps there's a better way to express this.

Reply:
Changed the wording. Let me know what you think.